src/components/pages/weather/elements/WeatherTempCell.js

```diff
@@ -1,10 +1,13 @@
 import React, { Component } from 'react';
 
 export default class WeatherTempCell extends Component {
-  render() {
-    const averageTemp = this.props.dayTemperatureData.avg.toFixed(3);
-    const maxTemp = this.props.dayTemperatureData.max.toFixed(3);
-    const minTemp = this.props.dayTemperatureData.min.toFixed(3);
+  constructor(props) {
+    super(props);
+    this.temperatureCellClass = null;
+  }
+
```

Comment:
I don't think we need a constructor, `temperatureCellClass` is a constant which we define in render rather than a property of the class.